Source/controls/plrctrls.cpp

```diff
@@ -1426,12 +1415,10 @@ bool TryDropItem()
 {
 	const auto &myPlayer = Players[MyPlayerId];
 
-	cursmx = myPlayer.position.future.x + 1;
-	cursmy = myPlayer.position.future.y;
+	cursPosition = myPlayer.position.future + Displacement { 1, 0 };
 	if (!DropItemBeforeTrig()) {
 		// Try to drop on the other side
-		cursmx = myPlayer.position.future.x;
-		cursmy = myPlayer.position.future.y + 1;
+		cursPosition = myPlayer.position.future + Displacement { 0, 1 };

```

Comment:
I feel like some of these suggestions lead to endless refactoring 

Would be nice if we could focus just on the changes at hand.

Again, i don't disagree with the suggestion necessarily, but we can't solve every issue on a code block just because we are touching that code block for something else.

Reply:
There are places where this would not have been the semantically right choice, which means the places where it should be changed are simple to locate. To me it's like using int where size_t should have been used, it works but the semantics are wrong and it will be easily overlooked in the future.